db/schema.rb

```diff
@@ -66,15 +64,16 @@
     t.integer "sdg_2"
     t.integer "sdg_3"
     t.boolean "sdgs_apply", default: false, null: false
+    t.integer "covid19_related", default: 0
```

Comment:
The unrelated schema changes make it trickier to distinguish what's actually changed. 😬

Reply:
Aaaaarrrgh. That was a total accident. I hate when this happens.